lib/Core/Reflection/Inference/Variable.php

```diff
@@ -21,14 +21,14 @@
      */
     private $value;
 
-    private function __construct(string $name, Offset $offset, Value $value)
```

Comment:
This information in this class is now duplicated in `SymbolInformation`, remove?